connect-examples/oauth/java/src/main/java/com/squareup/oauthexample/OAuthHandler.java

```diff
@@ -20,12 +20,12 @@
 
 package com.squareup.oauthexample;
 
-import com.squareup.connect.ApiException;
```

Comment:
nit: We should make sure the steps at the top of this file are still relevant.

Reply:
do you mean the comments?

Reply:
Yeah, the comments at the top of the page